vmaas/errata.go

```diff
@@ -0,0 +1,101 @@
+package vmaas
+
+import (
+	"slices"
+
+	"github.com/pkg/errors"
+	"github.com/redhatinsights/vmaas-lib/vmaas/utils"
+)
+
+type ErrataDetails map[string]ErratumDetail
+
+type Errata struct {
+	ErrataList ErrataDetails `json:"errata_list"`
+	Type       []string      `json:"type"`
```

Comment:
it's not in spec but the original implementation omits `Type` in the response when it is empty